plugins/available/bash-my-aws.plugin.bash

```diff
@@ -0,0 +1,44 @@
+cite about-plugin
+about-plugin 'Bash My AWS'
+
+export BASH_MY_AWS="${BASH_MY_AWS:-$HOME/.bash-my-aws}"
+
+__bma_load() {
+  local force=$1
+  if [[ -d "$BASH_MY_AWS" ]] ; then
+    if [[ -z $force ]] || [[ ":$PATH:" != *":$BASH_MY_AWS/bin:"* ]] ; then
+      pathmunge "$BASH_MY_AWS/bin"
+      source "$BASH_MY_AWS/aliases"
+      source "$BASH_MY_AWS/bash_completion.sh"
+    fi
+  fi
+}
+
+install-bash-my-aws() {
+  about 'Install bash-my-aws'
+  group 'bash-my-aws'
+
+  if [[ ! -d "$BASH_MY_AWS" ]] ; then
+    git clone https://github.com/bash-my-aws/bash-my-aws.git $BASH_MY_AWS
+    echo "bash-my-aws successfully installed."
+  else
+    echo "bash-my-aws already installed."
+  fi
+
+  __bma_load
+}
```

Comment:
I believe that installing dependencies should not be the business of bash-it. Explicitly per this comment: https://github.com/Bash-it/bash-it/issues/1470#issuecomment-584556474

Reply:
Yes, agreed. I'd prefer to not have tool installations included in Bash-it.